integration/next_context_test.go

```diff
@@ -42,3 +42,36 @@ func TestSettingTagContext(t *testing.T) {
 	tasks = unmarshalTaskArray(t, output)
 	assert.Equal(t, tasks[0].Summary, "two", "setting -one as a context")
 }
+
+func TestSettingTagAndProjectContext(t *testing.T) {
```

Comment:
Strictly speaking, this test is not related to the bugfix, but we were missing a minimal example related to setting `project:` attributes.